doc/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## What issue is this change attempting to solve?
```

Comment:
I suggest we also require a link to an issue on GH. This is recommended strategy described in contribution guidelines.

Reply:
Do you suggest to change the text?
Or do you know how to update the pull requests template to make some fields "required"?

Reply:
I suggest changing the text. E.g.
> Link to the issue this PR resolves

Reply:
Done

Reply:
Actually, reading https://github.com/bookingcom/carbonapi/blob/master/CONTRIBUTING.md again we have: "Refer to relevant issue if it's there.".
My understanding is that we can have PRs without linked issues. In this case I would use the initial phrasing so we could: 
a) link to issue if it's there
b) explain in the words what is this change about

Reply:
I thought these fields in the PR are optional. So, if we require the issue link, it is by default optional. Isn't it?

Reply:
Yes.
But then if there is not issue to link to, we can still have a short explanation, even if it's using words instead of links.